haproxy pythond plugin doc fix when using the plugin `via_url` (#10322)
netdata/netdata: collectors/python.d.plugin/haproxy/README.md

Plugin can obtain data from url **OR** unix socket.

**Requirement:**

> Socket: MUST be readable AND writable by the `netdata` user.

Reviewer: @joelhans please review, i am not sure about uppercase letters 🤔

> Socket: MUST be readable AND writable by the `netdata` user.

Reviewer 2: I would prefer we not use uppercase for emphasis. I'll suggest we use no emphasis, but @ashish1099 if you believe emphasis is important for users reading this doc, you can use italics `_must_` or bold `**must**`.

```suggestion
Socket: Must be readable and writeable by the `netdata` user.
```

Reviewer 3: The casing by @ashish1099 follows the pattern established by RFCs (where `MUST`, `MAY`, `SHALL` etc. have particular meanings), and has the benefit over Markdown formatting that it works in plaintext (e.g. without a renderer) as well as in rendered form. Perhaps a compromise?

```suggestion
Socket: **MUST** be readable **AND** writable by the `netdata` user.
```

Reviewer: Why do you think we need to follow the pattern established by RFCs if the readme is not RFC?

Reviewer 2: The Netdata [style guide](https://learn.netdata.cloud/contribute/style-guide#capitalization) explicitly says to not use capitalization for emphasis in documentation. I know there are places there the rules are still not followed, but let's say those are "grandfathered" in. All _new_ changes should follow the style guide.

Reviewer 3: > Why do you think we need to follow the pattern established by RFCs if the readme is not RFC?

Simply to follow an already established convention. I wasn't aware of the style guide, but that should of course be followed.

> URL: MUST have `stats uri <path>` present in the haproxy config, otherwise you will 503 in the haprooxy logs, more info [here](https://github.com/netdata/netdata/issues/10321)

Reviewer 3: ```suggestion
URL: MUST have `stats uri <path>` present in the haproxy config, otherwise you
will get HTTP 503 in the haproxy logs, more info [here](https://github.com/netdata/netdata/issues/10321)
```
